pygmo/expose_problems_1.cpp

```diff
@@ -155,5 +156,11 @@ void expose_problems_1()
                  bp::make_function(lcast([](unconstrain &udp) -> problem & { return udp.get_inner_problem(); }),
                                    bp::return_internal_reference<>()),
                  generic_udp_inner_problem_docstring().c_str());
+    // WFG.
+    auto wfg_p = expose_problem_pygmo<wfg>(
+        "wfg", "__init__(prob_id = 1, dim_dvs = 5, dim_obj = 3, dim_k = 4)\n\nThe WFG problem.\n\n"
```

Comment:
You need to use the appropriate docstring from ``docstrings.hpp`` here, instead of repeating the documentation of the ``__init__()`` method.

Reply:
Yes I noticed this when I generated the docs.. I am now doing the same as dlzt basically